src/SuperstateToken.sol

```diff
@@ -585,11 +585,14 @@ contract SuperstateToken is ISuperstateToken, ERC20Upgradeable, PausableUpgradea
 
     /**
      * @notice The ```subscribe``` function takes in stablecoins and mints SuperstateToken in the proper amount for the msg.sender depending on the current Net Asset Value per Share.
+     * @param to The address where USTB will be deposited at
      * @param inAmount The amount of the stablecoin in
      * @param stablecoin The address of the stablecoin to calculate with
      */
-    function subscribe(uint256 inAmount, address stablecoin) external {
-        //@TODO: update function, add support for to address
+    function subscribe(address to, uint256 inAmount, address stablecoin) external {
+        AllowList allowList = AllowList(address(allowListV2));
```

Comment:
Why do you need to create this var? IAllowListV2 doesnt expose addressEntityIds? If it doesnt you should add it instead of doing this line

Reply:
Updated the interface and added separate funcs for subscribe and subscribe to https://github.com/superstateinc/ustb/pull/88/commits/f7e42448bf50d3aa058d73ca93b9fb0c255f9cc1